MapboxDirections/MBDirections.swift

```diff
@@ -107,8 +107,6 @@ open class Directions: NSObject {
     
     public typealias MatchCompletionHandler = (_ waypoints: [Tracepoint]?, _ routes: [Match]?, _ error: NSError?) -> Void
```

Comment:
If we keep the first argument, it should be named `tracepoints`, not `waypoints`. The second argument should be named `matches`, not `routes`.